index.js

```diff
@@ -181,13 +181,16 @@ export const Rels = {
 
 export const Classes = {
 	activities: {
+		activity: 'activity',
 		assigned: 'assigned',
 		assignmentActivity: 'assignment-activity',
 		complete: 'complete',
 		completion: 'completion',
 		discussionActivity: 'discussion-activity',
 		exempt: 'exempt',
 		feedbackDate: 'feedback-date',
+		fileActivity: 'file-activity',
+		incomplete: 'incomplete',
```

Comment:
This one surprises me, shouldn't `incomplete` be the absence of `complete`?

Reply:
Surprisingly not! There's always the `completion` class, and either `complete` or `incomplete`, depending on a thing I'll link you to (since this is a public repo).

Reply:
Seems a bit odd but /shrug